.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,3 +14,11 @@ Include a few sentences describing the overall goals for this PR (pull request).
 - [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
 - [ ] New unit tests have been added for core changes
 - [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#contributing-to-qgis) before each commit
+
+## Backport
+
+<!-- Bug fixes can concern more than a branch and require backporting. Appropriate labels allow to automatically open the pull request in the corresponding branch, easing the process. You are however responsible of your code and should ensure there's no regression beforehand. Indicate below the version(s) you want to backport to.
+--> 
+
+- [ ] I request the backport of the changes to the Regular Release
```

Comment:
Latest

Reply:
Aaaah!!! That's it! I was wondering while "Regular Release" abbrveiation was LR at https://qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule